include/seqan3/alphabet/composition/cartesian_composition.hpp

```diff
@@ -93,6 +177,136 @@ struct cartesian_composition :
     static constexpr rank_type value_size{(alphabet_size_v<first_alphabet_type> * ... *
                                                alphabet_size_v<alphabet_types>)};
 
+    /*!\name Default constructors, destructor and assignment
+     * \{
+     * \attention Please do not directly use the CRTP base class. The constructors
+     *            are only public for the usage in their derived classes (e.g.
+     *            seqan3::qualified).
+     */
+    cartesian_composition() = default;
+    constexpr cartesian_composition(cartesian_composition const &) = default;
+    constexpr cartesian_composition(cartesian_composition &&) = default;
+    constexpr cartesian_composition & operator =(cartesian_composition const &) = default;
+    constexpr cartesian_composition & operator =(cartesian_composition &&) = default;
+    ~cartesian_composition() = default;
+
+    //!\brief Construction from initialiser-list.
+    constexpr cartesian_composition(first_alphabet_type first, alphabet_types ... others) :
+        base_t{first, others...}
+    {}
+    //!\}
+
+    /*!\name Conversion constructors and assignment
```

Comment:
no second group

Reply:
But if the default constructors are private and the custom ones public I'll have to make two groups out of them, no?